contents/docs/libraries/android/index.mdx

```diff
@@ -17,25 +17,23 @@ features:
 It uses an internal queue to make calls fast and non-blocking. It also batches requests and flushes asynchronously, making it perfect to use in any part of your mobile app.
 
 import AndroidInstall from '../../integrate/_snippets/install-android.mdx'
+import AndroidSendEvents from '../../integrate/send-events/_snippets/send-events-ios.mdx'
 import AndroidIdentify from './\_snippets/identify.mdx'
-import AndroidCapture from './\_snippets/capture.mdx'
 
 ## Installation
 
 <AndroidInstall />
 
-## Making calls
+## Sending events
```

Comment:
Again, would be nice to call this "capturing events" as it matches the name of the method used.